config/defaults.json

```diff
@@ -638,5 +638,125 @@
     "version":        "",
     "parent_key":     "",
     "parent_value":   ""
+  },
+  {
+    "key":            "ldap_host",
```

Comment:
For ease-of-use, can you move these so that they are alphabetical order? I also noticed the new funding settings aren't in alphabetical order, but we can fix that elsewhere.

Reply:
They can be moved but it won't matter in our database unless you have a fresh install.